docs/state.md

```diff
@@ -21,10 +21,9 @@ AppStore
   },
   about: {
     brave: {
-      versionInformation: [{
-        name: string,
-        version: string
-      }] // used on about:brave. not persisted (removed on save)
+      versionInformation: {
+        [name]: string
+      } // map of property name to version. used on about:brave. not persisted (removed on save)
```

Comment:
the 'not persisted (removed on save)' comment was there before. it doesn't seem to be a problem if it is in session-store-1 though